src/plugins/irc-events/connection.js

```diff
@@ -10,6 +10,35 @@ module.exports = function(irc, network) {
 		text: "Network created, connecting to " + network.host + ":" + network.port + "..."
 	}));
 
+	irc.on("registered", function() {
+		if (network.irc.network.cap.enabled.length > 0) {
+			network.channels[0].pushMessage(client, new Msg({
+				text: "Enabled capabilities: " + network.irc.network.cap.enabled.join(", ")
+			}));
+		}
+
+		var delay = 1000;
+		var commands = network.commands;
+		if (Array.isArray(commands)) {
+			commands.forEach(function(cmd) {
+				setTimeout(function() {
+					client.input({
+						target: network.channels[0].id,
+						text: cmd
+					});
+				}, delay);
+				delay += 1000;
```

Comment:
This could be simplified a bit;

```js
const delay = 1000;
commands.forEach((text, idx) => {
    setTimeout(() => client.input({
          target: network.channels[0].id,
          text,
     }), delay * ++idx);
});
```